angr/misc/loggers.py

```diff
@@ -107,6 +107,7 @@ def format(self, record: logging.LogRecord):
         level = level.ljust(8 + len(level) - lvl_len)
         body: str = f"{level} | {self.formatTime(record, self.datefmt) : <23} | {name} | {message}"
         if record.exc_info:
+            # pylint: disable=no-value-for-parameter
```

Comment:
????????????????????????????????????????

Reply:
Otherwise pylint complained that I wasn't passing `value` or `tb`.

From the [docs](https://docs.python.org/3/library/traceback.html#traceback.format_exception) they are optional though: `traceback.format_exception(exc, /, [value, tb, ]limit=None, chain=True)`.

Though upon further inspection the function signature did change so I'll revert this to support the older signature.